pixart.js

```diff
@@ -1 +1,37 @@
 
+$(document).ready(function(){
+  //don't reload the page
+  event.preventDefault();
+
+  //What color do you want
+  var userColor;
+  var userColorChoose = function(){
+    userColor = $("#color-field").val();
+    event.preventDefault();
+  };
+  //put this color in the box
+  var getColor = function(varX){
+    userColor = $("#color-field").val();
+    $(varX).css("background", userColor);
+    event.preventDefault();
+  };
+  //get the color on button click
+  $("button").on("click", function(){
+    userColorChoose();
```

Comment:
Include the `event.preventDefault()` right here so that it's the first line of code that runs when the button is clicked.